src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -717,6 +717,7 @@ protected function getKernelParameters()
                 'kernel.bundles_metadata' => $bundlesMetadata,
                 'kernel.charset' => $this->getCharset(),
                 'kernel.container_class' => $this->getContainerClass(),
+                'kernel.container_build_time' => time(),
```

Comment:
Wait, this doesn't work: before this change, the container had always the same hash, thus was stored always in the same dir. Now that this param is time dependent - and since it's dumped into the container, this property is broken.

Reply:
Do you mean because kernel.container_build_time will be different for every request? That is indeed true but it’s only used in the dumper.

Reply:
Yes, that's what I mean.

Reply:
Changed it to just introduce a method that returns the build time. People who want to make it deterministic can hook their own getenv() magic in there.